### ISSUE TITLE
don't leave alt attribute empty

### ISSUE THREAD

**Issue by ahmadmshatat** [FACG5/EliteTeam#20]
https://github.com/FACG5/EliteTeam/blob/e9a96fb3d8f803aa9f2715e017d150814e1aa3ad/index.html#L132

**Comment by SallamAltanna** [comment]
Yes, you have to fill alt attribute to reach high accessibility in your website!
